pkg/apic/service.go

```diff
@@ -142,27 +142,10 @@ func (c *ServiceClient) checkReferencesToAccessRequestDefinition(ard string) int
 
 // DeleteAPIServiceInstanceWithFinalizers deletes an api service instance in central, handling finalizers
 func (c *ServiceClient) DeleteAPIServiceInstanceWithFinalizers(ri *v1.ResourceInstance) error {
+	log.DeprecationWarningReplace("DeleteAPIServiceInstanceWithFinalizers", "DeleteAPIServiceInstance")
```

Comment:
I'm guessing we still need to keep this method on the ServiceClient, right? If not I can remove it.

Reply:
Thinkint about it I am quite sure the SDK was the only thing to call it. Unless @vivekschauhan thinks different I am going to say blow it away.

Reply:
I will remove it